### DISCUSSION TITLE
Need a hotkey to restart connection

### DISCUSSION THREAD

**Discussion by coelhoaf** [foxglove/community#749] [Q&A] [answered]
My colleagues and I have to reopen the websocket connection to run a new simulation. Since the process of going through the dropdown menu is very tedious, we'd like to request a hotkey to restart the most recently opened connection.

**Comment by achim-k** [comment, marked as answer]
What do you mean with running a new simulation? On the serve side you start a new simulation process (websocket server stays running) and you want Studio to reset?

Are you using foxglove websocket? Maybe https://github.com/foxglove/studio/pull/5775 is relevant for you, it allows the server to send a reset message to Studio

**Reply by coelhoaf** [reply]
Thanks for your reply. Yes, I'd like to restart the simulation on the server side without having to restart websocket, but since this didn't seem to work I thought a hotkey to restart websocket would facilitate the process. But the PR you mention seems to solve this issue. Thanks!
